packages/framework/src/Component/Setting/Setting.php

```diff
@@ -33,6 +33,8 @@ class Setting
      */
     protected array $values;
 
+    protected bool $allValuesLoaded = false;
```

Comment:
do we use annotation for typehinted properties? If so, you are missing annotation here :slightly_smiling_face:

Reply:
Not everywhere... I'm not sure if we have a strict rule about this. But I have added the annotation.